docs/ref/applications.txt

```diff
@@ -332,12 +332,13 @@ application registry.
 
 .. currentmodule:: django
 
-.. function:: setup()
+.. function:: setup(set_script=True)
 
     Configures Django by:
 
     * Loading the settings.
     * Setting up logging.
+    * If ``set_script`` is True, setting the URL resolver script prefix.
```

Comment:
add: "to settings.FORCE_SCRIPT_NAME if defined, or '/' otherwise."
please add `.. versionchanged`